SpatialGDK/Extras/schema/actor_group_member.schema

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+package unreal;
+
+component ActorGroupMember {
```

Comment:
Good to add a comment about what this component represents. Same for the set member. Mostly as they have similar names.